src/region/Region.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types';
+import { useEffect, useState, useRef } from 'react';
+
+import { StyledRegion } from './Region.styles';
+
+const Region = (props) => {
+  const { children, borderBottom, style, className, dataTestId } = props;
```

Comment:
Lets remove the style prop - it was there for some inlining quick styles when we were converting away from old components, but we don't need it now!